server/fleetv2_http_api/models/module.py

```
@@ -2,11 +2,11 @@
 
 from typing import List, Dict  # noqa: F401
 
-from server.fleetv2_http_api.models.base_model import Model
-from server.fleetv2_http_api.models.device_id import DeviceId
-from server.fleetv2_http_api import util
+from fleetv2_http_api.models.base_model import Model
+from fleetv2_http_api.models.device_id import DeviceId
+from fleetv2_http_api import util
 
-from server.fleetv2_http_api.models.device_id import DeviceId  # noqa: E501
+from fleetv2_http_api.models.device_id import DeviceId  # noqa: E501
```

Ruff (0.8.2)

Redefinition of unused `DeviceId` from line 6

Remove definition: `DeviceId`

(F811)